pkg/utils/formatting.go

```diff
@@ -94,6 +94,10 @@ func getPadWidths(stringArrays [][]string) []int {
 	}
 	return slices.Map(lo.Range(maxWidth-1), func(i int) int {
 		return slices.MaxBy(stringArrays, func(stringArray []string) int {
+			// the `cancel` button has only one column so this avoids a crash if maxWidth > 2
```

Comment:
I reckon that this function should be able to safely assume all the rows have the same number of columns. I would instead update the code that adds the cancel option to the menu to ensure it has the same number of columns as other rows.

Reason being that we want this function to catch bugs from elsewhere in the code that cause mismatched columns sizes